media-libs/libsvgtiny/libsvgtiny-0.1.3.recipe

```diff
@@ -11,12 +11,13 @@ CHECKSUM_SHA256="cbdc024743a56ccacbfa6d6bbce135ab6d701d38609f4a1ac56c2842457e5df
 PATCHES="libsvgtiny-0.1.1.patchset"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	libsvgtiny = $portVersion
+	libsvgtiny$secondaryArchSuffix = $portVersion
 	"
 REQUIRES="
-	haiku
+	haiku$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
```

Comment:
You have to add the $secondaryArchSuffix for the devel libs as well.
Also, I think you have to increate the REVISION by 1.